packages/core/src/render3/node_manipulation.ts

```
@@ -132,8 +133,9 @@ export function createElementNode(
   if (isProceduralRenderer(renderer)) {
     return renderer.createElement(name, namespace);
   } else {
-    return namespace === null ? renderer.createElement(name) :
-                                renderer.createElementNS(namespace, name);
+    const namespaceUri = namespace !== null ? getNamespaceUri(namespace) : null;
```

I'm wondering if we should throw an error or `console.log` in `ngDevMode` when the namespace is provided, but not recognized? Otherwise, we're ignoring the namespace and it'd be hard to catch that.

---

@atscott FYI I'll proceed with the merge. If you think that the proposal above makes sense, let's have a followup PR :)

---

This should be the same behavior as before. But also, @pkozlowski-opensource believes this `else` block isn't even ever hit and should just be removed. I only added this logic for completeness.

---

We are going though an extra function here (the `getNamespaceUri` one) that may result in a `null` is a namespace is not recognized. Previously we would proceed with the `createElementNS` in that case.